types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Minimum TypeScript Version: 3.0
+import {Node} from 'unist'
+
+declare namespace mdastToString {}
+
+declare function mdastToString(node: Node): string
```

Comment:
a list of nodes is also supported

otherwise looks good!

Reply:
updated!